src/constructs/ses/index.ts

```diff
@@ -0,0 +1,47 @@
+import { Duration } from "aws-cdk-lib";
+import type { EmailIdentityProps } from "aws-cdk-lib/aws-ses";
+import { EmailIdentity, Identity } from "aws-cdk-lib/aws-ses";
+import { GuAppAwareConstruct } from "../../utils/mixin/app-aware-construct";
+import type { AppIdentity, GuStack } from "../core";
+import { GuCname } from "../dns";
+
+export interface GuEmailIdentityProps extends Omit<EmailIdentityProps, "identity">, AppIdentity {
+  domainName: string;
```

Comment:
Could we add some inline documentation here, commenting on the need to be a `gutools.co.uk` domain?